tests/unit/MDMMotionCurveTests.m

```diff
@@ -53,4 +53,43 @@ - (void)testSpringCurveDataWithMacro {
   XCTAssertEqualWithAccuracy(curve.data[MDMSpringMotionCurveDataIndexFriction], 0.3, 0.001);
 }
 
+- (void)testSystemModalMovementTimingCurveMatchesModalMovementTiming {
+  UIWindow *window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  window.rootViewController = [[UIViewController alloc] init];
```

Comment:
Nit: Please use a designated initializer here and below: `initWithNibName:bundle:`

Reply:
Done.